README.md

```diff
@@ -7,6 +7,7 @@ more recent surface isolation architecture. [This article](./documents/xaml-desi
 
 - [XAML designer extensibility migration](./documents/xaml-designer-extensibility-migration.md)
 - [Toolbox population](./documents/toolbox-population.md)
+- [XAML Suggested Actions Extensibility](./documents/xaml-suggested-actions/xaml-designer-suggested-actions-extensibility.md)

```

Comment:
The word XAML seems unnecessary here since we're in the "XAML Designer Extensibility" repo.
Also "Extensibility" should probably be lowercase. I can understand Suggested Actions being uppercase since it's a feature name.